modules/sample/src/main/resources/issues/issue264.yaml

```diff
@@ -0,0 +1,22 @@
+openapi: 3.0.1
+info:
+  version: 1.0.0
+paths:
+  /foo:
+    get:
+      operationId: getFoo
+      parameters:
+        - in: query
+          name: FooEnum
+          $ref: "#/components/schemas/FooEnum"
+          required: true
+      responses:
+        204:
+          description: OK
+components:
+  schemas:
+    FooEnum:
+      type: string
+      enum:
+        - foo-bar-1
+        - baz-quux-2-flurb
```

Comment:
You mentioned "not handling numbers as the initial character in a grouping", but what if the numbers are the initial character in the entire enumeration?

Reply:
Welp, regardless, it'll be an invalid name in Java at least.